spec/functional/resource/dsc_script_spec.rb

```diff
@@ -67,8 +67,7 @@ def delete_user(target_user)
     node = Chef::Node.new
     node.automatic['platform'] = 'windows'
     node.automatic['platform_version'] = '6.1'
-    node.automatic['kernel'][:machine] =
-      is_i386_process_on_x86_64_windows? ? :x86_64 : :i386
+    node.automatic['kernel'][:machine] = :x86_64  # Only 64-bit process is supported
```

Comment:
So isn't the key here that we only support running these specs on a 64-bit OS vs. a 64-bit process? The specs themselves have always been 32-bit to this point, but Chef (also 32-bit) shells out to 64-bit).

My note here is only about the comment, basically I think it's more accurate to say 64-bit OS instead of 64-bit process. Does that seem reasonable?